run.go

```diff
@@ -92,6 +92,12 @@ func run(context *cli.Context) error {
 		if err != nil {
 			return fmt.Errorf("Process state %s: %s", ps.String(), err)
 		}
+
+		// delete container's resources
+		err = delete(pod.GetAllContainers()[0].ID(), true)
```

Comment:
I would prefer:
```go
if err := delete(container.ID(), true); err != nil {
```
And notice that you could factorize upper in the code something like this:
```go
container := pod.GetAllContainers()[0]
```